src/Homely.AspNetCore.Mvc.Helpers/Extensions/IApplicationBuilderExtensions.cs

```diff
@@ -25,12 +25,11 @@ public static class IApplicationBuilderExtensions
                 throw new ArgumentException(nameof(title));
             }
 
-            application.UseSwagger(c => c.RouteTemplate = routePrefix + "/{documentName}/swagger.json");
-
+            application.UseSwagger();
+            
             application.UseSwaggerUI(c =>
             {
                 c.SwaggerEndpoint($"/{routePrefix}/{version}/swagger.json", title);
-                c.RoutePrefix = routePrefix;

```

Comment:
how come this was removed?

Reply:
![image](https://user-images.githubusercontent.com/899878/76712183-56486200-676a-11ea-83b8-e089baa3de12.png)

Reply:
ref: https://docs.microsoft.com/en-us/aspnet/core/tutorials/getting-started-with-swashbuckle?view=aspnetcore-3.1&tabs=visual-studio

Reply:
So we _want_ to serve the swagger at root now?

I had it at `/swagger` for a reason -> the 'root' is usually the endpoint which shows the build version etc.

Reply:
correct. that's what it's doing _now_ with the current code.

```
c.SwaggerEndpoint($"/{routePrefix}/{version}/swagger.json", title

/swagger/v1/swagger.json

```